lib/index.js

```diff
@@ -58,7 +60,8 @@ class Reaptcha extends Component<Props> {
     tabindex: 0,
     explicit: false,
     inject: true,
-    isolated: false
+    isolated: false,
+    hl: ''
```

Comment:
i'm not very familiar with `flow`, should this be `''` or `undefined`?
not sure about this 😵

Reply:
Can be both, but as I've made the `className` and `id` with `''` - keep it the same way.